src/app/pages/services/services.module.ts

```diff
@@ -61,7 +64,7 @@ import { routing } from './services.routing';
     ServiceWebdavComponent,
     ServiceS3Component,
   ],
-  providers: [UserService],
+  providers: [UserService, ServicesService],
```

Comment:
You should not need to add service to providers when `{ providedIn: 'root' }` is used.